tests/utils/test_migration_helpers.py

```diff
@@ -0,0 +1,131 @@
+import warnings
+from typing import Tuple
+
+import pytest
+
+from kartothek.utils.migration_helpers import (
+    deprecate_parameters,
+    deprecate_parameters_if_set,
+)
+
+message = "Prameter {parameter} is deprecated!"
```

Comment:
A couple of minor changes:
1) typo: `Prameter -> Parameter`
2) should we call it `MESSAGE` instead of `message` as it is a global variable? Otherwise, we could create a fixture for this like:
```python
@pytest.fixture
def message() -> str:
    return "Parameter {parameter} is deprecated!"
    
def test_deprecate_optional_parameter_if_set_multi_table(message: str) -> None:
    ...
```

Reply:
I will capitalize it as a constant, you are right.
We can not use fixtures, because the decorators are constructed when the file is parsed, before the actual function calls are done.

Reply:
Yeah, right, the decorates created in the test file itself.

Reply:
(FYI: This is also the reason, why I test the error cases with nested functions)